user-guide/Cloud_Platform/CloudAdminApp/Auditing.md

```diff
@@ -0,0 +1,12 @@
+---
+uid: Auditing
+---
+
+# Auditing
```

Comment:
maybe add a note on this page containing a list of what auditing is supported up until now?

Reply:
Yeah did not want to do that since then we need to update it here...